content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,28 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+### Pod Deletion Cost 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Using the `controller.kubernetes.io/pod-deletion-cost`, users can set a preference regarding 
+which pods to remove first when downscaling a ReplicSet.
+
+The annotation should be set on the pod, the expected type is `int32`. It represents the cost of
```

Comment:
```suggestion
The annotation should be set on the pod, and the expected type is `int32`. It represents the cost of
```

Also, not sure if we should call it `int32`, as it's go type, and users are writing json/yaml.

Reply:
I think we should mention the type because this annotation will likely be set by controllers not users manually setting it.